src/components/HorizontalScroller.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useRef, useEffect } from 'react'
+import styles from './HorizontalScroller.module.scss'
+
+export const HorizontalScroller = ({ children, showArrows, classes }) => {
```

Comment:
Don't forget to set a default for classes here so it can be optional `classes = {}`